src/sentry/api/serializers/rest_framework/sentry_app_installation.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+
+from rest_framework import serializers
+from rest_framework.serializers import Serializer, ValidationError
+from sentry.constants import SentryAppInstallationStatus
+
+
+class SentryAppInstallationSerializer(Serializer):
+    status = serializers.CharField()
+
+    def validate_status(self, new_status):
+        # make sure the status in in our defined list
+        try:
+            SentryAppInstallationStatus.STATUS_MAP[new_status]
+        except KeyError:
+            raise ValidationError(
+                'Invalid value for status. Valid values: {}'.format(
+                    SentryAppInstallationStatus.STATUS_MAP.keys(),
+                ),
+            )
+        # convert status to str for comparison
+        last_status = SentryAppInstallationStatus.as_str(self.instance.status)
```

Comment:
I'm guessing this will fail if we use this when creating a new `SentryAppInstallation` since `self.instance` would be `None`. We probably want to wrap this in a `if self.instance`.